src/ws-dropdown/ws-dropdown.js

```diff
@@ -312,14 +318,26 @@ export class WSDropdown extends Component {
     }
     switch (this.props.type) {
       case 'anchor':
-        return <a className="dropdown-trigger" onClick={() => this.open()}>{icon} {this.state.text}</a>;
+        return <a className={"dropdown-trigger" + this.disabledStyle()}
```

Comment:
I guess the linter will complain here that you have to use a template string instead. So it would finally look like this
```js
return <a
  className={`dropdown-trigger ${disabledStyle}`}
  onClick={() => this.open()}
>
  {icon} {this.state.text}
</a>;
```